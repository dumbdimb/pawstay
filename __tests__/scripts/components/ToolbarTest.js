jest.unmock('../../../src/scripts/components/Toolbar');

import React from 'react';
import {shallow} from 'enzyme';
import renderer from 'react-test-renderer';

import Toolbar from '../../../src/scripts/components/Toolbar';

describe('Toolbar', () => {
    let tree;

    beforeEach(() => {
        tree = shallow(<Toolbar/>);
    });

    it('renders correctly', () => {
        expect(renderer.create(<Toolbar/>).toJSON()).toMatchSnapshot();
    });
});