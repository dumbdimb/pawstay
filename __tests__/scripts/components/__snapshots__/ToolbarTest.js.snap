// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toolbar renders correctly 1`] = `
<div
  className="toolbar"
>
  <div
    className="toolbar__content"
  >
    <div
      className="toolbar__logo"
    />
    <div
      className="toolbar__menu-item"
    >
      about us
    </div>
    <div
      className="toolbar__menu-item"
    >
      our service
    </div>
    <div
      className="toolbar__menu-item"
    >
      contact us
    </div>
  </div>
</div>
`;
